CHANGELOG.md

```diff
@@ -1,11 +1,18 @@
 # Changelog
 
+## [2.0.1](https://pypi.org/project/django-staff-sso-client/2.0.1/) (2020-03-25)
```

Comment:
Can you please change it to `2.1.0`, the third number is a bugfix version of the major number

Reply:
done